build.py

```diff
@@ -974,7 +974,6 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap):
 RUN wget -O /tmp/boost.tar.gz \
         https://archives.boost.io/release/1.80.0/source/boost_1_80_0.tar.gz && \
     (cd /tmp && tar xzf boost.tar.gz) && \
-    cd /tmp/boost_1_80_0 && ./bootstrap.sh --prefix=/usr && ./b2 install && \
```

Comment:
May I ask you to briefly clarify where else `boost` is used and why we are keeping it here?

Reply:
We use the header-only boost libs (`span` in cache, `interprocess_lock` in python backend, `stacktrace` in server, and some others) in several places. `boost::filesystem` was the only one that needed a runtime shared library requiring this extra `bootstrap+install` step.

Reply:
Got it, thanks

Reply:
Is there still an external dependency on libboost? Or could these header-only libs be included in-tree?